### PR TITLE
move array to json

### PR DESCRIPTION
**Pull request by ThorstenBuss**
Das Einnahmen und das Ausgabenarray sind jetzt in einer .json-Datei.

### CODE REVIEW

File: einnahmen.js [hunk 1]
```diff
@@ -96,77 +22,85 @@ Highcharts.setOptions({
...
-  colorMap[parents[i]] = "#" + colorSeq[i];
-}
-for (var i = 0; i < data.length; i++) {
-  if (data[i]['parent'] == "0.0") {
-    data[i]['color'] = colorMap[data[i]["id"]];
-  } else if (data[i]['id'] == "0.0") {
-    data[i]['color'] = '#ededed';
-  }
-}
+fetch('einnahmen.json')
```
Comment (christian-heusel) [inline einnahmen.js:25]: There seems to be something going wrong.

File: ausgaben.js [hunk 2]
```diff
@@ -361,59 +23,85 @@ Highcharts.setOptions({
...
-    data: data,
-    allowDrillToNode: true,
-    cursor: 'pointer',
-    dataLabels: {
-      format: '{point.title}',
-      filter: {
-        property: 'innerArcLength',
-        operator: '>',
-        value: 40
+fetch('ausgaben.json')
```
Comment (christian-heusel) [inline ausgaben.js:26]: Here too.

**Review by christian-heusel — CHANGES_REQUESTED** [review]
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///home/chris/Documents/shared_projects/Finance-Visualizer/einnahmen.json. (Reason: CORS request not http).
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///home/chris/Documents/shared_projects/Finance-Visualizer/ausgaben.json. (Reason: CORS request not http).

TypeError: NetworkError when attempting to fetch resource. einnahmen.js:25:6
TypeError: NetworkError when attempting to fetch resource. ausgaben.js:26:6
```

**Comment by christian-heusel** [comment]
At the moment I dont really understand the motivation to use JSON here.
An extra `.js` file would be sufficient as described in #5.

File: einnahmen.js (hunk 1 above)
Comment (ThorstenBuss) [reply einnahmen.js:25]: fetch() will not work on your local machine, but it does work on the server.

**Review by christian-heusel — APPROVED** [review]

**Comment by christian-heusel** [comment]
After deploying to my NAS I can confirm :hankey: :snail: 

Feel free to merge your PR!

**Comment by christian-heusel** [comment]
Please do not forget to also update the [README.md](https://github.com/FachschaftMathPhys/Finance-Visualizer/blob/master/README.md)

**Comment by ThorstenBuss** [comment]
As @TheEbolaDoc  described in #5 , it is probably nicer to use an extra .js instead of a .json file.

**Pull request closed without merging**